sr_description/hand/xacro/thumb/thumb.urdf.xacro

```diff
@@ -4,34 +4,27 @@
   <xacro:include filename="$(find sr_description)/hand/xacro/thumb/thhub.urdf.xacro" />
   <xacro:include filename="$(find sr_description)/hand/xacro/thumb/thmiddle.urdf.xacro" />
   <xacro:include filename="$(find sr_description)/hand/xacro/thumb/thdistal.urdf.xacro" />
-  <!-- muscletrans is true for muscle transmission to be used, false to use motor transmission -->
-  <!-- bio is true for biotac finger or false for standard finger -->
-  <!-- bt_sp is true for BT_SP finger or false for standard finger -->
-  <!-- ubi is true for ubi finger or false for standard finger -->
-  <!-- eli is true for ellipsoid fingertip or false for standard finger -->
+  <!-- tip_sensor describes the sensor none, bio, bt_sp, ubi, eli -->
+  <!-- mid_sensor describes the sensor none, bio, ubi -->
+  <!-- prox_sensor describes the sensor none, standard, ubi -->
   <!-- reflect is either 1 (for right hand) or -1 (for left hand) -->
   <!-- prefix can be anything but usually is "r_" or  "l_" for right and left hands distinction -->
   <!-- parent is the parent link for the first joint -->
   <!-- is_lite is this the lite hand, if so lock J3 -->
   <xacro:macro name="thumb"
-  params="muscletrans bio bt_sp ubi eli reflect prefix parent is_lite">
+  params=" tip_sensor:=^|std mid_sensor:=^|std prox_sensor:=^|std reflect:=^ prefix:=^ parent is_lite">
+    <xacro:if value="${tip_sensor == 'bio'}">
+      <xacro:property name="mid_sensor" value="bio" />
```

Comment:
:+1:  I had not seen this corner case where `tip_sensor` and `mid_sensor` have different sensors. In the finger only `tip_sensor` is considered indeed for middle and distal phalanges, but thumb was expecting `tip_sensor` and `mid_sensor` to be identical when using 'bio'.